config/environments/development.rb

```diff
@@ -34,13 +34,13 @@
   # For devise setup
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
 
+  config.action_mailer.delivery_method = :smtp
+
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
   config.action_mailer.perform_caching = false
 
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
revert

Reply:
It was already written above so I removed it from here.